v23.1/data-types.md

```diff
@@ -30,6 +30,8 @@ Type | Description | Example
 [`STRING`](string.html) | A string of Unicode characters. | `'a1b2c3'`
 [`TIME`<br>`TIMETZ`](time.html) | `TIME` stores a time of day in UTC.<br> `TIMETZ` converts `TIME` values with a specified time zone offset from UTC. | `TIME '01:23:45.123456'`<br> `TIMETZ '01:23:45.123456-5:00'`
 [`TIMESTAMP`<br>`TIMESTAMPTZ`](timestamp.html) | `TIMESTAMP` stores a date and time pairing in UTC.<br>`TIMESTAMPTZ` converts `TIMESTAMP` values with a specified time zone offset from UTC. | `TIMESTAMP '2016-01-25 10:10:10'`<br>`TIMESTAMPTZ '2016-01-25 10:10:10-05:00'`
+[`TSQUERY`](tsquery.html) | A list of lexemes and operators used in [full-text search](full-text-search.html). | `'list' & 'lexem' & 'oper' & 'use' & 'full' & 'text' & 'search'`
```

Comment:
do we want to add "New in" to these? I tend to add to option tables, but maybe there is a reason not to here.